### ISSUE TITLE
Produce a "How to buy bitcoin for Euros using Bisq" guide

### ISSUE THREAD

**Issue by cbeams** [bisq-network/docs#21]
Per the ongoing discussion in Slack at https://bisq.slack.com/archives/C701A4NNS/p1513844704000273. Further details to come.

**Bounty:** 200 $BSQ

**Comment by cbeams** [comment]
Note that this work is closely related to the video being produced at bisq-network/youtube#1. We may want to do one first and follow up with the other, as opposed to doing them in parallel and risking divergent approaches. My feeling is that we probably want to do the video first, and then the doc, because the video has more natural constraints forcing it to make sense and be logical. I'm open though, and encourage everybody involved (@francewhoa, @flix1, @leomcy) to collaborate and figure out what makes the most sense.

**Comment by cbeams** [comment]
Closing this in favor of putting together a general purpose Getting Started Guide, where we can link to the currency-specific videos we've done.
